src/h2load.cc

```diff
@@ -963,6 +965,31 @@ std::vector<std::string> read_uri_from_file(std::istream &infile) {
 } // namespace
 
 namespace {
+// Called every second when rate mode is being used
+void second_timeout_cb(EV_P_ ev_timer *w, int revents) {
+  auto config = static_cast<Config *>(w->data);
+
+  auto nclients_per_worker = config->rate;
+  auto nreqs_per_worker = config->max_concurrent_streams * config->rate;
+
+  if (config->current_worker >= std::max(0., (config->seconds - 1.))) {
```

Comment:
Not sure why you use double value here, since config->current_worker and config->seconds are both integers (ssize_t).